crates/notary/server/README.md

```diff
@@ -82,16 +81,16 @@ docker run --init -p 127.0.0.1:7047:7047 -v <your folder path>:/root/.notary-ser
    notarization:
      max_sent_data: 4096
      max_recv_data: 16384
+   
+   ...
     ```
    
-2. Command-Line Interface (CLI) Arguments    - You can override configuration file settings using CLI arguments when starting the server. CLI arguments take precedence over the config file.
-   - Example:
+2. Command-Line Interface (CLI) Arguments - You can override configuration file settings using CLI arguments when starting the server. They take precedence over both the config file and Environment Variables arguments, e.g.
    ```shell
-   cargo run -- --port 8080 --tls-enabled false --log-level INFO --max-sent-data 2048
+   cargo run -- --port 8080 --tls-enabled false --log-level INFO
    ```
 
-3. Environment Variables can also be used to configure the server. They take precedence over both the config file and CLI arguments. The environment variables use the prefix `NOTARY_SERVER__` followed by the configuration path in uppercase, with double underscores as separators.
-   - Example: 
+3. Environment Variables can also be used to configure the server and take precedence over the config file. The environment variables use the prefix `NOTARY_SERVER__` followed by the configuration path in uppercase, with double underscores used for nested configuration such that `tls.enabled` in the config file, which will be `TLS__ENABLED` on CLI, e.g.
```

Comment:
```suggestion
3. Environment Variables - This can be used to configure all the server settings, where it will override the config file. It uses the prefix `NOTARY_SERVER__` followed by the configuration key(s) in uppercase. Double underscores are used in nested configuration keys, e.g. `tls.enabled` in the config file will be `NOTARY_SERVER__TLS__ENABLED`. E.g.
```